routes/users.routes.js

```diff
@@ -7,8 +7,13 @@ const multer = require("multer");
 const router = express.Router();
 const uploadController = new UploadController();
 
-
-router.post('/uploads', authMiddleware,uploadController.sendMQ);
+console.log('users.routes.js');
+router.post('/uploads',authMiddleware,uploadController.sendMQ, uploadController.test2);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Disabling cross-site request forgery (CSRF) protection can make your users vulnerable to attacks. Enable CSRF if your application can be accessed through a browser and another token mechanism isn&#39;t in place.

[Learn more](https://cwe.mitre.org/data/definitions/352.html)